### PR TITLE
Update configuration_big_bird.py

### PR DESCRIPTION
**Pull request by parsa-ra**
Fixing backtick alignments

# What does this PR do?

Fixes # (issue)

## Before submitting
- [x] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).
- [ ] Did you read the [contributor guideline](https://github.com/huggingface/transformers/blob/master/CONTRIBUTING.md#start-contributing-pull-requests),
      Pull Request section?
- [ ] Was this discussed/approved via a Github issue or the [forum](https://discuss.huggingface.co/)? Please add a link
      to it if that's the case.
- [ ] Did you make sure to update the documentation with your changes? Here are the
      [documentation guidelines](https://github.com/huggingface/transformers/tree/master/docs), and
      [here are tips on formatting docstrings](https://github.com/huggingface/transformers/tree/master/docs#writing-source-documentation).
- [ ] Did you write any new necessary tests?

## Who can review?

Anyone in the community is free to review the PR once the tests have passed. Feel free to tag
members/contributors who may be interested in your PR.

### CODE REVIEW

**Review by patil-suraj — COMMENTED** [review]
Thanks for fixing this! Could you run `make style` and push again, that will fix the failing test.

**Comment by parsa-ra** [comment]
@patil-suraj Sry for the late response, Any idea on what would be the problem? 
```bash
make style
black examples tests src utils
Skipping .ipynb files as Jupyter dependencies are not installed.
You can fix this by running ``pip install black[jupyter]``
All done! ✨ 🍰 ✨
1489 files left unchanged.
isort examples tests src utils
Skipped 1 files
make autogenerate_code
make[1]: Entering directory '/home/parano/github/dev/transformers'
  File "setup.py", line 212
    entries = "\n".join([f'    "{k}": "{v}",' for k, v in deps.items()])
                                            ^
SyntaxError: invalid syntax
make[1]: *** [Makefile:22: deps_table_update] Error 1
make[1]: Leaving directory '/home/parano/github/dev/transformers'
make: *** [Makefile:64: style] Error 2

```

File: src/transformers/models/big_bird/configuration_big_bird.py [hunk 1]
```diff
@@ -100,6 +98,8 @@ class BigBirdConfig(PretrainedConfig):
         BigBirdModel(configuration)
 
         >>> # Accessing the model configuration >>> configuration = model.config
```
Comment (patil-suraj) [inline src/transformers/models/big_bird/configuration_big_bird.py:100]:
```suggestion
        >>> # Accessing the model configuration
        >>> configuration = model.config
```

File: src/transformers/models/big_bird/configuration_big_bird.py [hunk 2]
```diff
@@ -88,9 +88,7 @@ class BigBirdConfig(PretrainedConfig):
     Example:
 
     ```python
-
-    ```
-
+    
         >>> from transformers import BigBirdModel, BigBirdConfig
 
         >>> # Initializing a BigBird google/bigbird-roberta-base style configuration >>> configuration =
```
Comment (patil-suraj) [inline src/transformers/models/big_bird/configuration_big_bird.py:94]:
```suggestion
        >>> # Initializing a BigBird google/bigbird-roberta-base style configuration
```

File: src/transformers/models/big_bird/configuration_big_bird.py [hunk 3]
```diff
@@ -100,6 +98,8 @@ class BigBirdConfig(PretrainedConfig):
         BigBirdModel(configuration)
```
Comment (patil-suraj) [inline src/transformers/models/big_bird/configuration_big_bird.py:98]:
```suggestion
        >>> configuration = BigBirdModel(configuration)
```

File: src/transformers/models/big_bird/configuration_big_bird.py [hunk 4]
```diff
@@ -100,6 +98,8 @@ class BigBirdConfig(PretrainedConfig):
...
 
         >>> # Accessing the model configuration >>> configuration = model.config
+    ```
+
```
Comment (patil-suraj) [inline src/transformers/models/big_bird/configuration_big_bird.py:102]:
```suggestion
```

**Review by patil-suraj — CHANGES_REQUESTED** [review]

**Comment by patil-suraj** [comment]
> @patil-suraj Sry for the late response, Any idea on what would be the problem?
> 
> ```shell
> make style
> black examples tests src utils
> Skipping .ipynb files as Jupyter dependencies are not installed.
> You can fix this by running ``pip install black[jupyter]``
> All done! ✨ 🍰 ✨
> 1489 files left unchanged.
> isort examples tests src utils
> Skipped 1 files
> make autogenerate_code
> make[1]: Entering directory '/home/parano/github/dev/transformers'
>   File "setup.py", line 212
>     entries = "\n".join([f'    "{k}": "{v}",' for k, v in deps.items()])
>                                             ^
> SyntaxError: invalid syntax
> make[1]: *** [Makefile:22: deps_table_update] Error 1
> make[1]: Leaving directory '/home/parano/github/dev/transformers'
> make: *** [Makefile:64: style] Error 2
> ```

Make sure you have python>=>=3.6.0 and the required versions black. You can install the correct style deps by using

`pip install transformers["quality"]`

**Comment by parsa-ra** [comment]
@patil-suraj The problem arises since `make` by default assumes that when we call `python` we mean `python2`, not `python3` (At least in my system, ubuntu 20.04.4 LTS), may be we should change the `python` commands in the `setup.py` and `Makefile` to `python3`, since making and `alternative` or `alias` for python3 is not actually a very good idea (as can be seen they are not presenting the same kind of functionalities ... ) We might also ask for the comments of maintainers ...

**Pull request closed without merging**